docs/workflow.rst

```diff
@@ -3,255 +3,182 @@
-running beast_production_wrapper:
+To create a physics model grid with 5 subgrids:
 
   .. code-block:: console
 
-     $ run beast_production_wrapper.py
+     $ beast.tools.run.create_physicsmodel --nsubs=5
 
-The first thing it does is use datamodel_template.py to create a
-datamodel.py file.  This will be imported as needed in the functions
-called by the wrapper.  As noted above, five of the datamodel fields
-will be updated, so ensure that the other fields in
-datamodel_template.py have the desired values.
 
-The wrapper will proceed through each of the functions below.  At
-three points, you will need to manually run things independently of
-the wrapper.  It will not continue running subsequent functions until
-it finds that the necessary steps have been taken.
+*********************
+Artificial Star Tests
+*********************
 
-  * Creating ASTs (if a fake star catalog doesn't exist)
-  * running the batch trimming scripts
-  * running the batch fitting scripts
+The observation model is based on artificial star tests (ASTs).  The BEAST
+selects SEDs from the physics model grid.  For each band, the range of fluxes
+in the model grid is split into bins (default=40, set by datamodel.ast_n_flux_bins),
+and models are randomly selected.  The model is retained if there are fewer than
+the set number of models (default=50, set by datamodel.ast_n_per_flux_bin) in
+each of the relevant flux bins.
 
-Once you have completed each of these, run the wrapper again.  It will
-skip past the steps that it has already processed, and resume at the point
-where you left off.  In the case of the batch scripts, if you only
-partially completed them, it will re-generate new scripts for the
-remaining trimming/fitting (and tell you which ones are new), and
-pause again.
+Then the stars are placed within the image.  For each of the options below, each
+SED may be placed once (for the toothpick model) or multiple times (for the
+truncheon model), as set by datamodel.ast_realization_per_model.
 
-Note of warning: if you are using this wrapper for multiple fields,
-check that the proper version of datamodel.py is in place before
-running the batch trimming/fitting scripts.  For instance, if you have
-recently used the wrapper to do part of the processing for field_A,
-and you want to start the batch fitting script for field_B, re-run the
-wrapper for field_B to make sure that datamodel.py refers to the
-information for field_B.
+  * Option 1 (datamodel.ast_source_density_table is set):
+    For each source density or background bin, randomly place the SEDs
+    within pixels of that bin.  Repeat for each of the bins.
+  * Option 2 (datamodel.ast_source_density_table = None):
+    Randomly choose a star from the photometry catalog, and place the
+    artificial star nearby.  Repeat until all SEDs have been placed.
 
+.. code-block:: console
 
-****
-Data
-****
+   $ beast.tools.run.make_ast_inputs --flux_bin_method=True
```

Comment:
Probably should be "./beast/tools/run/make_ast_inputs --flux_bin_method=True".

Reply:
resolved per discussion below